packages/components/src/Auth.tsx

```diff
@@ -241,7 +241,8 @@ function NetworkInput({
   )
 }
 
-function UrlInput({ setUrl }: { setUrl: Dispatch<string> }) {
+function UrlInput({ urlInputRef }: { urlInputRef: React.RefObject<HTMLInputElement> }) {
+  const [url, setUrl] = useState<string | undefined>(process.env["REACT_APP_URL"] || "")
```

Comment:
The UrlInput is only shown if shown process.env["REACT_APP_URL"] evaluates to false, so we could directly initialize url to "".